apps/website/src/components/admin/short-links/ShortLink.tsx

```diff
@@ -0,0 +1,143 @@
+import { useState } from "react";
+import Link from "next/link";
+import type { inferRouterOutputs } from "@trpc/server";
+import { trpc } from "@/utils/trpc";
+import {
+  Button,
+  dangerButtonClasses,
+  LinkButton,
+} from "@/components/shared/Button";
+import { ModalDialog } from "@/components/shared/ModalDialog";
+import { Headline } from "@/components/admin/Headline";
+import { Panel } from "@/components/admin/Panel";
+import IconPencil from "@/icons/IconPencil";
+import IconTrash from "@/icons/IconTrash";
+import { env } from "@/env/index.mjs";
+import type { AppRouter } from "@/server/trpc/router/_app";
+type RouterOutput = inferRouterOutputs<AppRouter>;
+type ShortLink = RouterOutput["adminShortLinks"]["getLinks"][number];
+
+type LinkProps = {
+  shortLink: ShortLink;
+  onError: (error: string) => void;
+  onUpdate: () => void;
+};
+
+function ShortLinks({ shortLink, onError, onUpdate }: LinkProps) {
+  const deleteMutation = trpc.adminShortLinks.deleteLink.useMutation({
+    onError: (error) => onError(error.message),
+    onSettled: () => onUpdate(),
+  });
+  const clicks = trpc.adminShortLinks.getClicks.useQuery();
+  const clicksMap = new Map(clicks.data?.map((c) => [c.id, c.clicks]));
+
+  return (
+    <>
+      <tr className="border-b border-gray-700">
+        <td className="p-1">
+          <Button size="small" width="auto">
+            {"✅"}
+          </Button>
+        </td>
+        <td className="w-1/2 p-1">
+          <div className="flex flex-col gap-0.5">
+            <div className="text-xl">{shortLink.label}</div>
+            <div className="flex flex-col gap-1">
+              <Link
+                href={`/l/${shortLink.slug}`}
+                target="_blank"
+                className="flex"
+              >
+                <div className="mr-1.5">Public Link:</div>
+                <div className="underline">
+                  {env.NEXT_PUBLIC_SHORT_BASE_URL + "/l/" + shortLink.slug}
```

Comment:
Please use the [`getShortBaseUrl`](https://github.com/alveusgg/alveusgg/blob/main/apps/website/src/utils/short-url.ts) helper here please?

Reply:
didn't know we had it